examples/blend/main.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2023 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"image"
+	_ "image/png"
+	"log"
+	"math"
+
+	"golang.org/x/image/colornames"
+	"golang.org/x/image/font/inconsolata"
+
+	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/examples/resources/images/blend"
+	"github.com/hajimehoshi/ebiten/v2/text"
+)
+
+const (
+	screenWidth  = 780
+	screenHeight = 600
+)
+
+// mode is a blend mode with description.
+type mode struct {
+	blend ebiten.Blend
+	name  string
+}
+
+// Game is a canvas for drawing blend mode tiles.
+type Game struct {
+	source       *ebiten.Image
+	dest         *ebiten.Image
+	offscreen    *ebiten.Image
+	tileSize     float64
+	gridW, gridH float64
+	tileSpacing  float64 // Horizontal and vertical margin between tiles.
+	textSpacing  float64 // Additional vertical margin for text.
+	modes        []mode
+}
+
+func NewGame() *Game {
+	g := &Game{
+		source: loadImage(blend.Source_png),
+		dest:   loadImage(blend.Dest_png),
+		gridW:  4, gridH: 3,
+		tileSpacing: 64, textSpacing: 16,
```

Comment:
This is just my preference but put one member for one line

```
		source: loadImage(blend.Source_png),
		dest:   loadImage(blend.Dest_png),
		gridW:  4,
		gridH: 3,
		tileSpacing: 64,
		textSpacing: 16,
```

Reply:
Ok

Reply:
Fixed